terraform/workspace-variables/staging.tfvars

```diff
@@ -5,9 +5,9 @@ app_environment = "staging"
 # Gov.UK PaaS
 paas_api_url = "https://api.london.cloud.service.gov.uk"
 paas_space_name = "early-careers-framework-staging"
-paas_postgres_service_plan = "tiny-unencrypted-11"
+paas_postgres_service_plan = "small-11"
 paas_app_start_timeout = "180"
 paas_app_stopped = false
 paas_web_app_deployment_strategy = "blue-green-v2"
 paas_web_app_instances = 1
-paas_web_app_memory = 512
\ No newline at end of file
+paas_web_app_memory = 8192
```

Comment:
What are these based on? Or did you just pick what you deemed suitable?

Reply:
Asked @cfq. I think JC picked a number months ago

Reply:
Are you just curious, or do you disagree with the number?

Reply:
Just curious

Reply:
Will we have just one web server then?

Reply:
I think at least initially, yes. We're only going to have 2 users for the next month... Hopefully ramp up will be quite gradual, so we can be reactive to load. It's something that's easy to change, so I think we're ok to start small